client/rest-high-level/src/test/java/org/elasticsearch/client/SearchIT.java

```diff
@@ -835,6 +877,174 @@ public void testRenderSearchTemplate() throws IOException {
         assertToXContentEquivalent(expectedSource, actualSource, XContentType.JSON);
     }
 
+    public void testExplainSimple() throws IOException {
```

Comment:
I would prefer not using Simple in a method name, although it's a test. Nothing is really simple :) Call it testExplain?

Reply:
Yeah definitly agree xD